common/changes/@itwin/core-frontend/markschlosser-draco-security-fix_2025-08-20-17-57.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/core-frontend",
+      "comment": "",
```

Comment:
i would call out this change in the changelog even if its an internal detail
as well as mention it in nextver.md